assets/javascripts/discourse/pre-initializers/extend-category-for-voting.js.es6

```diff
@@ -35,13 +35,14 @@ function initialize(api) {
     },
     { priority: -100 }
   );
+  api.addCategorySortCriteria("votes");
 }
 
 export default {
   name: "extend-category-for-voting",
   before: "inject-discourse-objects",
   initialize(container) {
-    withPluginApi("0.8.4", api => {
+    withPluginApi("0.8.30", api => {
```

Comment:
This will mean that none of the plugin JS will execute for older versions of Discourse (e.g. stable branch). Could the `api.addCategorySortCriteria` be done in a different `withPluginApi` call?